core/runtime/src/main/java/io/quarkiverse/langchain4j/guardrails/OutputGuardrail.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkiverse.langchain4j.guardrails;
+
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.memory.ChatMemory;
+import dev.langchain4j.rag.AugmentationResult;
+
+public interface OutputGuardrail {
```

Comment:
We'll definitely need some Javadoc for this 😉

Reply:
Done.